Realm/Tests/RLMMultiProcessTestCase.h

```diff
@@ -0,0 +1,31 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMTestCase.h"
+
+@interface RLMMultiProcessTestCase : RLMTestCase
+// if true, this is running the main test process
+@property (nonatomic, readonly) bool isParent;
+
+// spawn a child process running the current test and wait for it complete
+// returns the return code of the process
+- (int)runChildAndWait;
+@end
+
+#define RLMRunChildAndWait() \
```

Comment:
The macro is so that errors are reported on the line where RLMRunChildAndWait() appears rather than inside the helper method.